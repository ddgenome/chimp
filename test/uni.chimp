# unimolecular surface reaction
mechanism "uni.mech"
## parameter input
parameter "uni.par"
## simple task
task "uni.task"
