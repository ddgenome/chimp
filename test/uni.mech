# a unimolecular surface reaction mechanism
# adsorption / desorption of A
A + @ -> k_constant(A_Aads) <- k_arrhenius(A_Ades, E_Ades) @A;
# surface reaction
@A -> k_arrhenius(A_srf, E_srf) <- k_arrhenius(A_srr, E_srr) @B;
# adsorption / desorption of B
@B <- k_constant(A_Bads) -> k_arrhenius(A_Bdes, E_Bdes) B + @;
