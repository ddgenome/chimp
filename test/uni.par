# parameter input file for simple mechanism
A_Aads	1.0e-2	# molec/Pa/site/s
A_Ades	1.0e14	# molec/site/s
E_Ades	9.1e4	# J/mol
A_srf	1.0e9	# molec/site/s
E_srf	6.5e4	# J/mol
A_srr	1.0e8	# molec/site/s
E_srr	8.0e4	# J/mol
A_Bdes	8.0e13	# molec/site/s
E_Bdes	9.2e4	# J/mol
A_Bads	8.0e-3	# molec/Pa/site/s
