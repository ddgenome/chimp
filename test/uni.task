# -*- text -*-
# unimolecular surface reaction task input
begin model uni
  output "uni.out"
  begin integrator kmc
    size 30
    scale 1.0e15
    surface_file "uni.surface"
    begin state
      begin quantity
	p[A] = 1.0e5		# Pa
      end quantity
      begin output
	1.0e-5 1.0e-1 (1.0e0 1.0e1)
      end output
      begin reactor batch
	temperature 4.2e2	# K
	volume 1.0e-5		# m^3
	rate_numerator molecules
	rate_denominator sites
	fluid_quantity pressure
      end reactor
    end state
  end integrator
end model
