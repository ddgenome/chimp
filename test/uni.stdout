chimp 0.1
Copyright (C) 2004 David Dooling <banjo@users.sourceforge.net>

There is ABSOLUTELY NO WARRANTY; see file COPYING for details.
This is free software, and you are welcome to redistribute it
under certain conditions; see file COPYING for details.

Report bugs to http://sourceforge.net/projects/chimp/.
chimp: performing task uni.par... completed; 0u 0s 0w
chimp: performing task uni... completed; 2.72u 0.02s 2.73w
